docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+RUN apt-get clean && apt-get update && apt-get install -y locales tzdata
+RUN locale-gen en_US.UTF-8
+RUN update-locale
+RUN mkdir /usr/src/app
+# Set your timezone
+RUN ln -fs /usr/share/zoneinfo/Europe/Moscow /etc/localtime && dpkg-reconfigure -f noninteractive tzdata
+WORKDIR /usrc/src/app
+RUN apt-get update \
+  && apt-get install -y build-essential libpam0g-dev apt-utils cpanminus \
+  && apt-get autoremove -y \
+  && rm -r /var/cache/apt/archives/* \
+  && rm -r /var/lib/apt/*
+RUN cpanm Text::CSV App::mojopaste
+ENV PASTE_DIR /usr/src/app/data
+ENV MOJO_CONFIG /usr/src/app/mojopaste.conf
```

Comment:
Why not just use `/app`?

Reply:
Also, why are there no ENTRYPOINT an CMD?

Reply:
Dir will by only in container. It does not matter where it will be inside the container.
It my by and just app

What do you mean ENTRYPOINT? For what? For dir? 
Can show me expample

Reply:
If the data is inside the container, then after it is deleted, the data will be lost.

Therefore, there is this line - `CMD="$CMD -v $PWD:/usr/src/app"`. 

Therefore, it does not matter where the data lies inside the container.

Reply:
Also, why are there no ENTRYPOINT an CMD?
I understood about this. I'll do it